apps/cli/src/main.ts

```diff
@@ -273,27 +249,27 @@ export const executeMainThread = async () => {
         return executeCliCommand(() => handleWhoAmICommand({ printer }));
       },
     )
-    .command(
-      "build",
-      "build the JavaScript engine codemod",
-      (y) =>
-        y.option("source", {
-          alias: "s",
-          type: "string",
-          description: "path to the codemod to be built",
-        }),
-      async (args) => {
-        const { executeCliCommand, printer } =
-          await initializeDependencies(args);
-
-        return executeCliCommand(() =>
-          handleBuildCliCommand({
-            printer,
-            source: args.source ?? process.cwd(),
-          }),
-        );
-      },
-    )
+    // .command(
```

Comment:
Why was it commented?

Reply:
to be removed, this command is no longer needed.